File: dev-superior-algoritmo-e-logica-de-programacao/c/exercicios/menor_de_tres.c
#include <stdio.h>

int main() {
    int num1, num2, num3, menor;

    printf("Primeiro valor:");
    scanf("%d", &num1);

    printf("Segundo valor:");
    scanf("%d", &num2);

    printf("Terceiro valor:");
    scanf("%d", &num3);

    if ((num1 < num2) && (num1 < num3)) {
        menor = num1;
    } else if (num2 < num3) {
        menor = num2;
    } else {
        menor = num3;
    }

    printf("MENOR = %d", menor);

    return 0;
}

File: nelio-alves-algoritmo-e-logica-de-programacao/python/exercicios/menor_de_tres.py
num1: int; num2: int; num3: int; menor: int

num1 = int(input("Primeiro valor: "))
num2 = int(input("Segundo valor: "))
num3 = int(input("Terceiro valor: "))

if num1 < num2 and num1 < num3:
	menor = num1
elif num2 < num3:
	menor = num2
else:
	menor = num3

print(f"MENOR = {menor}")